CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ All notable changes to this project will be documented in this file.
 -   \[INTERNALS\] Use `FunctionDefArgument` to store all argument specific properties.
 -   \[INTERNALS\] Reduce carbon footprint by avoiding unnecessary CI testing.
 -   \[INTERNALS\] Automatise PR labelling and review progress prompts.
+-   Default to linking python dynamically instead of statically
```

Comment:
Should we not capitalize `Pyhon`? Why is the spell checker not complaining?

Reply:
Fixed 97ae2e5f